sorbet/rails-rbi/models/user/habtm_manages_collections.rbi

```diff
@@ -77,7 +57,6 @@ module User::HABTM_ManagesCollections::CustomFinderMethods
 end
 
 class User::HABTM_ManagesCollections < ActiveRecord::Base
-  include User::HABTM_ManagesCollections::GeneratedAttributeMethods
```

Comment:
Did you run `bundle exec srb rbi hidden-definitions`?

Reply:
I did not - but I do have the pre-push hook, which does not include this.